app/views/maps/index.html.erb

```diff
@@ -105,8 +105,6 @@
                 attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
     }).addTo(Mapknitterunique) ; 
 
-    var hash = new L.Hash(Mapknitterunique);    
```

Comment:
Can you post the error ? How did you figure out it is Hash library which is causing error ? Thank you 😄 !

Reply:
@sagarpreet-chadha you are right. So rather we should place is down because I guess this is creating a problem and removing it, it was working properly.